### PR TITLE
added sem tabs

### LINKED ISSUE #28: Create tabs to switch between different semesters
# User Story
As a student, I want to be able to create timetables for multiple semesters

# Description
Create a tabs section that lets the user choose the semester they want to create a timetable for. 

# Additional Details 
https://material-ui.com/components/tabs/
The TabPanel should provide the selected semester as a prop to its child.
Semester information will be obtainable from https://github.com/VikeLabs/scheduler-back-end/issues/12

# Acceptance Criteria
- [ ] Switching tabs should change the semester chosen for creating a timetable
- [ ] Tests must be created and pass

### PR DESCRIPTION
**Pull request by jbenning97**
* Added a theme object
* Added a navbar (mostly empty, we can add as we go)
* Added semester tabs (called SemTabs)

### CODE REVIEW

File: src/molecules/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Toolbar from '@material-ui/core/Toolbar';
+import { StyledNavBar } from './styles';
+
+const NavBar: React.FC = () => {
+  return (
+    <StyledNavBar position="static">
+      <Toolbar>
+        <h2>VikeLabs / </h2>
```
Comment (kerfootj) [inline src/molecules/NavBar/NavBar.tsx]: Can we change these to both be the same size? I think it looks best when they're both h3's but I'll get someone else to take a look at it as well. 

Can we reduce the margin a little bit? I think the navbar is just a bit too tall. 

Also can you link to https://vikelabs.ca/

File: src/molecules/SemTabs/SemTabs.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import AppBar from '@material-ui/core/AppBar/AppBar';
+import Tabs from '@material-ui/core/Tabs/Tabs';
+import Tab from '@material-ui/core/Tab/Tab';
+import Theme from '../../atoms/Theme/Theme';
```
Comment (kerfootj) [inline src/molecules/SemTabs/SemTabs.tsx]: absolute import

File: src/molecules/SemTabs/SemTabs.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```
Comment (kerfootj) [inline src/molecules/SemTabs/SemTabs.tsx]: I would appreciate if you renamed this to SemesterTabs but you don't have to. Personally I find longer names easier read but that might just be me.

**Review by kerfootj — CHANGES_REQUESTED** [review]

**Review by kerfootj — APPROVED** [review]

**Review by kerfootj — CHANGES_REQUESTED** [review]
You have a merge conflict now

**Comment by jbenning97** [comment]
@kerfootj Should be g2g now.

**Review by kerfootj — APPROVED** [review]

**Pull request merged**